integrations/tensorflow/bindings/python/pyiree/tf/support/tf_test_utils.py

```diff
@@ -276,7 +276,13 @@ def _check_same(ref, tar, rtol, atol):
             ref.dtype, tar.dtype)
         return False
       if np.issubdtype(ref.dtype, np.floating):
-        return np.allclose(ref, tar, rtol=rtol, atol=atol)
+        same = np.allclose(ref, tar, rtol=rtol, atol=atol)
+        if not same:
+          logging.error(
+              "Floating point difference between ref and tar was too large. "
+              "Max absolute difference: %s, rtol: %s, atol: %s",
```

Comment:
shouldn't you print the max relative different as well? the rtol might be the one being violated.

Reply:
Good idea.